### ISSUE TITLE
Characters aren't available immediately after making their achievements public on the Lodestone

### ISSUE THREAD

**Issue by JamesDonnelly** [ApkalluFalls/apkallufalls.com#7]
Characters are not immediately updated on XIVDB following changes to Lodestone settings. This is causing some confusion as users cannot find their characters on Apkallu Falls until XIVDB recognises this change.

There are a few parts to this:

- [ ] Investigate whether it's possible to update XIVDB's character reference via their API.
- [ ] Implement feature on the main website.
- [ ] Make it clear that this feature is available to avoid confusion.

**Comment by JamesDonnelly** [comment]
This is now resolved as of #5
